style.css

```diff
@@ -0,0 +1,262 @@
+  width: auto;
+  height: auto;
+  font-family: Roboto, sans-serif;
+  color: #c8cdd4;
+}
+
+.navbar header {
+  position: relative;
+  width: 375px;
+  height: 92px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #191765;
+}
+
+main {
+  width: 375px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: #191765;
+  background-image: url(./assets/images/eclipes.png);
+  background-repeat: no-repeat;
+  background-position-x: -200px;
+}
+
+header a {
+  font-family: Roboto, sans-serif;
+  font-size: 18px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  text-align: center;
+  color: #fff;
+  text-decoration: none;
+  padding-left: 24px;
+  padding-right: 24px;
+}
+
+.hamburger {
+  background-color: #191765;
+  border: #191765;
+  padding-right: 24px;
+}
+
+.hamburger :hover {
+  background-color: #e5e5e5;
+}
+
+.hamburger:focus {
+  outline: none;
+  box-shadow: 0 0 0 3px #7d73fc;
+}
+
+.hamburger:active {
+  background-color: #e5e5e5;
+  border-color: #acb7c3;
+  color: #acb7c3;
+}
+
+.profile-container h1 {
+  font-family: Roboto, sans-serif;
+  font-size: 48px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 72px;
+  letter-spacing: 0.3700000047683716px;
+  text-align: center;
+  color: #7d73fc;
+}
+
+.profile-container h2 {
+  font-family: Roboto, sans-serif;
+  font-size: 32px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 44px;
+  letter-spacing: 0;
+  text-align: center;
+  color: #edecff;
+}
+
+.profile-container p {
+  font-family: Roboto, sans-serif;
+  font-size: 20px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 28px;
+  letter-spacing: 0;
+  text-align: center;
+  color: #c8c8d4;
+  padding-left: 24px;
+  padding-right: 24px;
+  padding-top: 40px;
+}
+
+.skill-stack {
+  height: 462.1505432128906px;
+  width: 303px;
+  left: 12px;
+  top: 235px;
+  border-radius: 0;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+}
+
+.skill-stack h3 {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  text-align: left;
+  color: #9490ff;
+}
+
+li {
+  list-style: none;
+}
+
+.skill-stack li {
+  font-family: Roboto, sans-serif;
+  font-size: 15px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 24px;
+  letter-spacing: 0;
+  text-align: left;
+}
+
+.social-media {
+  height: 104px;
+  width: 200px;
+  left: 87.5px;
+  top: 1092.1505126953125px;
+  border-radius: 0;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.social-media-icon {
+  height: 40px;
+  width: 200px;
+  left: 0;
+  top: 0;
+  border-radius: 0;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  color: #fff;
+  padding-top: 48px;
+}
+
+.experience-title {
+  padding-top: 83.85px;
+}
+
+.experience-title h2 {
```

Comment:
- [ ] Kindly follow the given design template and fix this title's spacings (margin and padding)
_Please keep in mind that you should follow the original design's typography, font size, font weight, colors, images, and spacings in order to provide a professional, consistent, and good-looking portfolio.
### **Your design**
![Screenshot](https://user-images.githubusercontent.com/47625407/121764937-d0bb8980-cb47-11eb-9728-e3f91e62da35.png)
### **[Original design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10)**
![Screenshot1](https://user-images.githubusercontent.com/47625407/121764944-e0d36900-cb47-11eb-8ac2-d86ba23e2456.png)